### DISCUSSION TITLE
Dependency can't be resolved after add local dependency

### DISCUSSION THREAD

**Discussion by LintaoAmons** [mfussenegger/nvim-jdtls#594] [Q&A]
![image](https://github.com/mfussenegger/nvim-jdtls/assets/95092244/20019078-2dce-4de1-b4cc-d8a3bad43959)

And I also tried open the project by vscode and verified that it's can be resolved

**Comment by LorenzoBettini** [comment]
Have you tried with `:JdtUpdateConfig`?
